Library/Homebrew/global.rb

```diff
@@ -13,7 +13,7 @@
 # Only require "core_ext" here to ensure we're only requiring the minimum of
 # what we need.
 require "active_support/core_ext/object/blank"
-require "active_support/core_ext/numeric/time"
+require "active_support/core_ext/string/filters"
```

Comment:
This was a transitive `require` that we rely on in a few places (in particular, `String#remove`).